Android/Wear/nuget/Xamarin.Android.Wear.nuspec

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.Android.Wear</id>
+    <title>Xamarin Android Wear Libraries</title>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <summary>Xamarin.Android Bindings for Wear Libraries</summary>
+    <description>
+      Xamarin.Android Bindings for Wear Libraries
+    </description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=868403</projectUrl>
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=868402</licenseUrl>
+    <dependencies>
+      <group targetFramework="MonoAndroid80">
+        <dependency id="Xamarin.Android.Support.Annotations" version="26.1.0.1" />
+        <dependency id="Xamarin.Android.Support.Compat" version="26.1.0.1" />
+        <dependency id="Xamarin.Android.Support.Core.UI" version="26.1.0.1" />
+        <dependency id="Xamarin.Android.Support.Percent" version="26.1.0.1" />
+        <dependency id="Xamarin.Android.Support.v7.RecyclerView" version="26.1.0.1" />
+      </group>
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="nuget/Xamarin.Android.Wear.targets" target="build/MonoAndroid80" />
+    <file src="externals/support_wearable/proguard.txt" target="proguard/proguard.txt" />
```

Comment:
Is there a specific reason for this to be in the root and not in the build directory? It may be better to not put too much in the root just in case they do something to nuget to either make this mean something, or do some validation with where stuff lives.

Reply:
Perhaps that's where it should go instead.  That way we can stick MonoAndroid80 in front of it too...